src/pretix/control/forms/event.py

```diff
@@ -528,6 +556,36 @@ def __init__(self, *args, **kwargs):
             (a, {"title": a, "data": v}) for a, v in get_fonts().items()
         ]
 
+        # create "virtual" fields for better UX when editing <name>_asked and <name>_required fields
+        self.virtual_keys = []
+        for asked_key in [key for key in self.fields.keys() if key.endswith('_asked') and key[:-5] + 'required' in self.fields]:
+            required_key = asked_key[:-5] + 'required'
```

Comment:
This is probably a matter of taste, but I'd find ``asked_key.rsplit('_', 1)[0]`` way more readable than ``asked_key[:-5]``. It's longer, but it expresses my *intent* and I don't need to think about why there is a 5 and count characters. What do you think?

Reply:
I wasn’t to happy about that magic number either, so totally for removing it! I thought about adding `suffix = '_asked'` and use `asked_key[:-len(suffix)].` but I wasn’t convinced. Your suggestion totally matches the intent of „divide string in two parts at the last _ and use the first part". It might be me, but I had to think a second about the second paramter to resplit. What about `asked_key[:asked_key.rindex('_')]` ? Either way is fine.

Reply:
You're probably missing a ``-`` in your approach, but other than that, I don't really care. Your choice :=